nautobot/users/migrations/0001_initial.py

```diff
@@ -1,119 +1,115 @@
-# Generated by Django 3.1.3 on 2021-02-20 08:07
+# Generated by Django 3.1.7 on 2021-03-10 06:59
 
-import uuid
 from django.conf import settings
 import django.contrib.auth.models
+import django.contrib.auth.validators
 import django.contrib.postgres.fields
+import django.core.serializers.json
 import django.core.validators
 from django.db import migrations, models
 import django.db.models.deletion
+import django.utils.timezone
+import uuid
 
 
 class Migration(migrations.Migration):
 
     initial = True
 
     dependencies = [
-        ("contenttypes", "0002_remove_content_type_name"),
         ("auth", "0012_alter_user_first_name_max_length"),
-        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("contenttypes", "0002_remove_content_type_name"),
```

Comment:
This seems like it might be of concern. The fact that the swappable dependency was removed?

Reply:
@jathanism So my understanding of `migrations.swappable_dependency()` (and after looking at the [source](https://github.com/django/django/blob/d40402cfb023801f0d83f19747e30b13096e3636/django/db/migrations/migration.py#L216-L218)) is that it generates a migration dependency for the user model which may or may not have actually been swapped. The original migration had an actual dependency on the `contrib.auth` user model but because we create our own now, I would think it would cause some sort of circular dependency here.

In any event, `makemigrations` is responsible for this specific change.